code/modules/mob/living/carbon/alien/humanoid/emote.dm

```diff
@@ -8,8 +8,23 @@
 //	if(findtext(act,"s",-1) && !findtext(act,"_",-2))//Removes ending s's unless they are prefixed with a '_'
 //		act = copytext(act,1,length(act))	//seriously who the fuck wrote this
 	var/muzzled = is_muzzled()
+
+	var/on_CD = 0
```

Comment:
This actually does work as the var is changed by the CD handler function in mob/emote. This var doesn't handle the CD itself, it just holds the result of the actual handler function from the parent.

I also tested this in game and it works exactly as expected